### DISCUSSION TITLE
Cannot Login

### DISCUSSION THREAD

**Discussion by zelkadude** [btcpayserver/btcpayserver#5501] [Technical Support] [answered]
I Made a BTCpay server account last night, i tried logging in today but it simply wont work.

**Comment by mp21-org** [comment]
PSA: Our mainnet demo server database passed away. (RIP 2018-2023)  We will create a new server!

https://twitter.com/BtcpayServer/status/1727191336162623910

**Comment by mp21-org** [comment, marked as answer]
PSA: Mainnet demo server database recovered!

https://twitter.com/BtcpayServer/status/1727347484597555623
